sdk/include/ndk/i386/asm.h

```diff
@@ -471,6 +471,17 @@ Header Name:
 #endif
 #define EFLAGS_USER_SANITIZE                    0x3F4DD7
 
+
+//
+// HYPERVISOR 
```

Comment:
The description doesn't need to be all-caps, and can be a bit more elaborate. E.g. "Hypervisor enlightenment flags", or "Hypervisor Enlightenment Definitions" as is used in kd386.template.h